### DISCUSSION TITLE
Libre2 + miaomiao3 without nfc support: is it possible?

### DISCUSSION THREAD

**Discussion by dagord** [NightscoutFoundation/xDrip#2417] [Q&A] [answered]
Hello,
my girlfriend is happily using Libre1 + Miaomiao3 with xDrip+ for several months.
Now our n.h.s. is switching to Libre2 so I'm trying to understand how this works; as I read I need the modified Freestyle Librelink app to initialize the sensor, but the phone doesn't have NFC support and I understand it's mandatory for activation, isn't it?
Is there a way to use it with the Miaomiao3 as she used with Libre1?
For the moment, she's using the standalone Freestyle reader

**Comment by tzachi-dar** [comment, marked as answer]
She can use libre2 with miaomiao3 using the oop2 approach. She will have to keep wearing the miaomiao3 all the time though.

**Reply by dagord** [reply]
Thanks for your reply.
I know about Oop2 and I tried to get info online, but I wasn't able to find any "official" documentation... only some guides.
But as far as I understand, it requires the initial scan to be done via NFC protocol. Am I wrong?

**Reply by PhilTU** [reply]
You have to start the sensor and the reader will do that, if sufficiently recent firmware even a L1 reader starts L2 sensor.

The NFC for starting a sensor is a one off if using a transmitter devicecould be done with another random phone.

**Reply by dagord** [reply]
Hello, thank you for your answer.
In the meanwhile I got it working simply enabling the "OOP algorithm" on xDrip, and leaving other settings untouched.
I forgot to close the issue... sorry

**Reply by Navid200** [reply]
@dagord 
No need to close a discussion.  
But, it's great to confirm that your problem/question has been addressed.
Thanks
